core/app/help.go

```diff
@@ -23,6 +23,45 @@ import (
 	"github.com/google/gapid/core/app/flags"
 )
 
+// MakeDocs
+func MakeDoc(ctx context.Context, name string){
+	filename := name + ".md"
+	w, _ := os.Create(filename)
+	globalVerbs.selected = nil
+	fmt.Fprintln(w, "**Note**: Autogenerated documentation, do not modify!")
```

Comment:
Here I would add in the comment how this doc is generated, e.g.:

**Note**: Autogenerated documentation via `gapit make_doc`, do not modify by hand.

Reply:
This should be a comment not visible to the user. Is that possible with MD? If not, it should be moved to the bottom or at least after the title.